2-mandatory/2-financial-times.js

```diff
@@ -14,24 +17,43 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let fewestWordTitle = allArticleTitles[0];
+    for (const article of allArticleTitles) {
+        if (article.length < fewestWordTitle.length) {
+            fewestWordTitle = article;
+        }
+    }
+    return fewestWordTitle;
```

Comment:
This is a good solution with simple use of an iterated array. You can of course also do it using `forEach` as well. If you are happy to investigate other solutions using inline functions (like you did with `filter` above), you can have a read of how the `reduce` function works that can be used to implement things like miniimums, maximums or sums: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce